moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -517,7 +517,7 @@ void plan_execution::PlanExecution::successfulTrajectorySegmentExecution(const E
   if (index < plan->plan_components_.size() && plan->plan_components_[index].trajectory_ &&
       !plan->plan_components_[index].trajectory_->empty())
   {
-    if (!isRemainingPathValid(*plan, std::make_pair<int>(index, 0)))
+    if (!isRemainingPathValid(*plan, std::make_pair<int>(static_cast<int>(index), 0)))
```

Comment:
This should work as well (and is cleaner in my opinion, because a pair should get two template args, not only one):
```suggestion
    if (!isRemainingPathValid(*plan, std::make_pair(static_cast<int>(index), 0)))
```